Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/FileExportRuleSet.java

```diff
@@ -376,13 +390,18 @@ public int compareTo(Rule that) {
         List<Long> evaluate(long dataSourceId) throws ExportRulesException {
             try {
                 SleuthkitCase db = Case.getCurrentCaseThrows().getSleuthkitCase();
+                ResultSet resultSet = null;
                 try (SleuthkitCase.CaseDbQuery queryResult = db.executeQuery(getQuery(dataSourceId))) {
-                    ResultSet resultSet = queryResult.getResultSet();
+                    resultSet = queryResult.getResultSet();
```

Comment:
Is it feasible to include the ResultSet in the try so it is closed via AutoCloseable?

Reply:
Apparently it is when putting each as its own line of code in the parens, thank you for helping me learn this.